internal/handlers/accounts_v3_users_handler.go

```diff
@@ -65,7 +66,55 @@ func AccountsV3UsersHandler(w http.ResponseWriter, r *http.Request) {
 
 		r := usersToV3Response(u.Users)
 
+		client.CloseSdkConnection()
+
 		sendJSON(w, r.Responses)
+	case keycloakModule:
+		orgID := chi.URLParam(r, "orgID")
```

Comment:
Would it make sense to abstract some of this out if possible since it's shared between the types of providers/modules?